src/libs/ReportActionsUtils.js

```diff
@@ -138,6 +150,17 @@ function getParentReportActionInReport(childReportID, parentReportID) {
     return _.find(allReportActions[parentReportID], (reportAction) => reportAction && `${reportAction.childReportID}` === `${childReportID}`);
 }
 
+/**
+ * Get the details linked to the IOU reportAction
+ *
+ * @param {Object} reportAction
+ * @returns {Object}
+ */
+function getLinkedTransaction(reportAction = {}) {
+    const transactionID = lodashGet(reportAction, ['originalMessage', 'IOUTransactionID'], '');
+    return allTransactions[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`] || {};
+}
+
```

Comment:
Should we move this to transactionUtils? We already have the allTransactions there